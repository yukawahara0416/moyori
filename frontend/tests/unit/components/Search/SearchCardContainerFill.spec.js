import { shallowMount } from '@vue/test-utils'
import Component from '@/components/Search/SearchCardContainerFill.vue'

let wrapper
let propsData

beforeEach(() => {
  propsData = {
    spots: []
  }

  wrapper = shallowMount(Component, {
    propsData
  })
})

afterEach(() => {
  wrapper.destroy()
})

describe('props', () => {
  it('spots', () => {
    expect(wrapper.props().spots).toStrictEqual(propsData.spots)
    expect(wrapper.props().spots instanceof Array).toBe(true)
  })
})

describe('template', () => {
  it('snapshot', () => {
    expect(wrapper.vm.$el).toMatchSnapshot()
  })
})
