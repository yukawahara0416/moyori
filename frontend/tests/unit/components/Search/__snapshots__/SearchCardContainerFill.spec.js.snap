// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`template snapshot 1`] = `
<v-row-stub
  align="center"
  class="mx-0"
  justify="center"
  tag="div"
>
  <v-col-stub
    cols="10"
    tag="div"
  >
    <p>
      キーワード検索できます
    </p>
     
    <p>
      現在地に戻ることができます
    </p>
     
    <p>
      地図のサークル内のカフェを検索できます
    </p>
     
    <v-img-stub
      class="white--text align-end"
      options="[object Object]"
      position="center center"
      src=""
      transition="fade-transition"
    />
  </v-col-stub>
</v-row-stub>
`;
